public/components/ResultsComparison.vue

```diff
@@ -109,8 +109,12 @@ export default Vue.extend({
       return this.viewTypeModel;
     },
 
-    isTableView(): boolean {
-      return this.viewType === TABLE_VIEW;
+    /* Check if any items display on the table have a weight property. */
+    hasWeight(): boolean {
+      const data = this.includedTableDataItems ?? [];
+      return data.some(item =>
+        Object.values(item).some(variable => variable.hasOwnProperty("weight"))
```

Comment:
Just for reference, lodash has a nice impl of this:  https://lodash.com/docs/4.17.15#get